barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -60,6 +60,7 @@ struct AcirFormat {
     std::vector<poly_triple_<curve::BN254::ScalarField>,
                 ContainerSlabAllocator<poly_triple_<curve::BN254::ScalarField>>>
         constraints;
```

Comment:
Should we rename this to `poly_triple_constraints`?

Reply:
Done